packages/core/android/src/main/kotlin/com/datadog/reactnative/FrameRateProvider.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import com.facebook.react.modules.core.ChoreographerCompat
+
+internal class FrameRateProvider(
+    reactFrameRateCallback: ((Double) -> Unit)?
+) {
+    private val frameCallback: FpsFrameCallback = FpsFrameCallback(reactFrameRateCallback)
+
+    fun start() {
+        frameCallback.reset()
+        frameCallback.start()
+    }
+
+    fun stop() {
+        frameCallback.stop()
+    }
+}
+
+internal class FpsFrameCallback(
+    private val reactFrameRateCallback: ((Double) -> Unit)?
+) :
+    ChoreographerCompat.FrameCallback() {
+    private var mChoreographer: ChoreographerCompat? = null
+
+    private var mLastFrameTime = -1L
```

Comment:
```suggestion
    private var choreographer: ChoreographerCompat? = null

    private var lastFrameTime = -1L
```

just to align the naming convention

Reply:
I agree